students/jemielity_kamil/lesson_01_basics/total_cost.py

```diff
@@ -0,0 +1,9 @@
+import math
+dollars = int(input('How many dollars: '))
+cents = int(input('How many cents: '))
+number_of_cupcakes = int(input('Number of cupcake: '))
+
+wallet = str(dollars) + '.' + str(cents)
```

Comment:
Check what happens if cents = 6.
It might be better solution to count total price in cents instead of using float value